jenkins/spark-tests.sh

```diff
@@ -24,36 +24,35 @@ nvidia-smi
 ARTF_ROOT="$WORKSPACE/jars"
 MVN_GET_CMD="mvn org.apache.maven.plugins:maven-dependency-plugin:2.8:get -B \
     -Dmaven.repo.local=$WORKSPACE/.m2 \
-    $MVN_URM_MIRROR -DremoteRepositories=$URM_URL \
-    -Ddest=$ARTF_ROOT"
+    $MVN_URM_MIRROR -Ddest=$ARTF_ROOT"
 
 rm -rf $ARTF_ROOT && mkdir -p $ARTF_ROOT
 # maven download SNAPSHOT jars: cudf, rapids-4-spark, spark3.0
-$MVN_GET_CMD \
+$MVN_GET_CMD -DremoteRepositories=$CUDF_REPO \
     -DgroupId=ai.rapids -DartifactId=cudf -Dversion=$CUDF_VER -Dclassifier=$CUDA_CLASSIFIER
-$MVN_GET_CMD \
+$MVN_GET_CMD -DremoteRepositories=$PROJECT_REPO \
     -DgroupId=com.nvidia -DartifactId=rapids-4-spark_$SCALA_BINARY_VER -Dversion=$PROJECT_VER
-$MVN_GET_CMD \
-    -DgroupId=com.nvidia -DartifactId=rapids-4-spark-udf-examples -Dversion=$PROJECT_VER
-$MVN_GET_CMD \
-    -DgroupId=com.nvidia -DartifactId=rapids-4-spark-integration-tests_$SCALA_BINARY_VER -Dversion=$PROJECT_VER
+$MVN_GET_CMD -DremoteRepositories=$PROJECT_TEST_REPO \
```

Comment:
Here we diff `PROJECT_VER/PROJECT_REPO` & `PROJECT_TEST_VER/PROJECT_TEST_REPO`.
We only release rapids-4-spark.jar(`PROJECT_VER`) onto Sonatype repo(`PROJECT_REPO`).
rapids-4-spark-udf-examples-`SNAPSHOT`.jar/rapids-4-spark-integration-tests-`SNAPSHOT`.jar(`PROJECT_TEST_VER`) are on internal repo(`PROJECT_TEST_REPO`)